test-cli.py

```diff
@@ -420,7 +422,7 @@ def test(ctx):
     check_contains(['run "echo hello"'], run_command([cl, 'info', '-f', 'args', uuid]))
     check_equals('hello', run_command([cl, 'cat', uuid+'/stdout']))
     # block
-    uuid2 = check_contains('hello', run_command([cl, 'run', 'echo hello', '--tail'])).split('\n')[0]
+    uuid2 = check_contains('hello', run_command([cl, 'run', 'echo hello', '--tail']))
```

Comment:
If it's not used just delete it

On Fri, Feb 12, 2016 at 8:11 PM, Andrew <notifications@github.com> wrote:

> In test-cli.py
> <https://github.com/codalab/codalab-cli/pull/314#discussion_r52822656>:
>
> > @@ -420,7 +422,7 @@ def test(ctx):
> >      check_contains(['run "echo hello"'], run_command([cl, 'info', '-f', 'args', uuid]))
> >      check_equals('hello', run_command([cl, 'cat', uuid+'/stdout']))
> >      # block
> > -    uuid2 = check_contains('hello', run_command([cl, 'run', 'echo hello', '--tail'])).split('\n')[0]
> > +    uuid2 = check_contains('hello', run_command([cl, 'run', 'echo hello', '--tail']))
>
> It seems like uuid2 isn't being used at all, so I think I was actually
> going to go ahead and delete the assignment as well but forgot about it.
> Would you rather I leave it in?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codalab/codalab-cli/pull/314/files#r52822656>.
>